### PR TITLE
Adding sauce to the build

### PR DESCRIPTION
**Pull request by CodeBaboon**
Added saucelabs configuration settings to the Travis build.

### CODE REVIEW

**Comment by ryantmer** [comment]
dammit IE.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-- 4
-sudo: false
-addons:
-  firefox: "47.0.1"
-before_script:
-- npm run test:lint:js && npm run test:lint:wc
-script:
-- xvfb-run wct
-after_success:
-- '[ "$TRAVIS_SECURE_ENV_VARS" = "true" ] && [ "x$TRAVIS_TAG" != "x" ] && npm run publish:cdn'
```
Comment (ryantmer) [inline .travis.yml]: Isn't this still needed for publishing to the CDN?

**Comment by ryantmer** [comment]
Seems like CI hung on Sauce

```
08 Aug 17:46:23 - GET https://saucelabs.com/rest/v1/apalaniuk/tunnels?full=1: HTTP response code indicated failure.
08 Aug 17:46:23 - Response: {"error": "Not authorized"}.
```

**Comment by ryantmer** [comment]
Whoop! LGTM :+1: 

Also seems like it did run for both /pr and /push, so that's legit

**Comment by apalaniuk** [comment]
Okay, so this works (ish) now. Travis seems to need to have concurrent builds limited to 1 for the moment which would be nice to remove.

**Comment by ryantmer** [comment]
@apalaniuk Yeah that's a limitation of the travis.org accounts I believe. Unfortunate, but ah well.

**Pull request merged**